src/containers/quiz/quiz-info/QuizInfo.styles.ts

```diff
@@ -12,7 +12,15 @@ const styles = {
       flexDirection: 'column',
       alignItems: 'flex-start',
       gap: theme.spacing(2)
-    }
+    },
+    marginBottom: theme.spacing(2)
+  },
+  infoWrapperNew: {
+    display: 'flex',
+    flexDirection: 'row',
+    alignItems: 'center',
+    gap: '10px',
```

Comment:
```suggestion
    gap: theme.spacing(1.25),
```
Should work here as well.